ignite/cmd/plugin_test.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 
-	"github.com/ignite/cli/ignite/config"
+	config "github.com/ignite/cli/ignite/config/plugins"
```

Comment:
It would be good for consistency to use the following here and everywhere else:

```go
	pluginsconfig "github.com/ignite/cli/ignite/config/plugins"
```

Reply:
https://github.com/ignite/cli/pull/3184/commits/e6858996520488ad1297add95c1003835fca0cd4